src/arith_uint256.cpp

```diff
@@ -146,13 +146,19 @@ double base_uint<BITS>::getdouble() const
 template <unsigned int BITS>
 std::string base_uint<BITS>::GetHex() const
 {
-    return ArithToUint256(*this).GetHex();
+    base_blob<BITS> b;
+    for(int x = 0; x < this->WIDTH; ++x)
```

Comment:
nit: Might be good for new code to format properly.

Also the commit title contains a newline:

```
The subject line of commit hash c612bdf509c6534581d68a25d614545980bceb01 is followed by a non-empty line. Subject lines should always be followed by a blank line.
```

```suggestion
    for (int x = 0; x < this->WIDTH; ++x) {
```

Reply:
I fixed the formatting. About the commit message, thanks to Qt Creator... doing that without my consent. Should I rebase to fix this or do we let it go?

Reply:
> I fixed the formatting.
> OK. I just noticed the linter isn't happy about this. Seems like I'll have to rebase.

While you are at it, the fix for the formatting needs to be squashed into the same commit that either introduced the code.

Reply:
Alright. Rebased and squashed. Everything now is one commit.